src/Transport/HttpTransport.php

```diff
@@ -108,25 +119,26 @@ public function send(Event $event): ?string
         } else {
             try {
                 $promise->wait();
-            } catch (\Exception $exception) {
-                return null;
+            } catch (\Throwable $exception) {
+                return $event->getId();
```

Comment:
```suggestion
                // ignore transport issues
```

In the future we could create a hook here to log issues.

Reply:
See: #907

Reply:
I reverted this change as thinking again about it it's not a regression since the method could already return `null` values and it makes a lot of sense if something fails to not return the event ID